### PR TITLE
Catch some missing translations

### PR DESCRIPTION
**Pull request by terotik**

### CODE REVIEW

File: public/locales/fi/common.json [hunk 1]
```diff
@@ -76,5 +76,11 @@
   "sort-descending": "Lajittele laskevassa järjestyksessä",
   "skip-to-main-content": "Siirry pääsisältöön",
   "outcome-sub-nodes": "Alasektorit",
-  "outcome-sub-nodes-secondary": "Alasektorit"
+  "outcome-sub-nodes-secondary": "Alasektorit",
+  "all-settings": "Kaikki asetukset",
+  "display": "Näytetään",
+  "hide-calculation": "Näytä laskenta",
```
Comment (woodwoerk) [inline public/locales/fi/common.json:82]: Duplicate content from `show-calculation`?

Comment (woodwoerk) [inline public/locales/fi/common.json:82]: Duplicate content from show-calculation?

**Review by woodwoerk — APPROVED** [review]
Please check that duplicate Finnish translation, otherwise looks good

File: public/locales/fi/common.json (hunk 1 above)
Comment (terotik) [reply public/locales/fi/common.json:82]: Good catch 👍

**Pull request merged**